### PR TITLE
[PUPIL-864] Add quiz hints to quizzes

### PR DESCRIPTION
**Pull request by BTitterington**
## Description

Music year: 2019

Adds hints to quizzes.

## How to test

1. Go to https://deploy-preview-2615--oak-web-application.netlify.thenational.academy/pupils/programmes/maths-primary-year-4/units/calculation-and-conversion-of-measures/lessons/use-efficient-strategies-and-common-measure-conversions-to-solve-problems-in-a-range-of-contexts/starter-quiz
2. Click on the hint in the bottom left
3. You should see a hint and be able to close it.

## Screenshots

image

## Checklist

- [x] Added or updated tests where appropriate
- [x] Manually tested across browsers / devices
- [x] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [x] Does this PR update a package with a breaking change

### CODE REVIEW

File: src/__tests__/__helpers__/IntersectionObserverMock.ts
Comment (kimon-satan) [inline src/__tests__/__helpers__/IntersectionObserverMock.ts:1]: Ah funny I just came across this and took the different approach of exporting the mocks from oak components. We probably should settle on one method but lets just merge both for the time being

**Review by kimon-satan — APPROVED** [review]

**Comment by RBrownsett** [comment]
@BTitterington tested - functionally no issues and works as epected. The accessibility checker did pick up an issue, though I'm not sure if its a red herring, stating that the aria-describedby element does exists but the target for the reference does not - this is for the hint button - any ideas? 

image

More info here:

https://equalizedigital.com/accessibility-checker/broken-aria-reference/

**Comment by BTitterington** [comment]
@RBrownsett should be good to check accessibility again now.

**Review by RBrownsett — APPROVED** [review]
Approved - retested accessibility has now been fixed

**Pull request merged**